cli_tools_e2e_test/gce_ovf_import/scripts/ovf_import_test_ubuntu_3_disks.sh

```diff
@@ -29,63 +77,43 @@ function fail {
   FAILURES+="TestFailed: $message"$'\n'
 }
 
-# Check network connectivity.
-function check_connectivity {
+function check_metadata_connectivity {
   status "Checking metadata connection."
-  ping -q -c 2 metadata.google.internal
-  if [[ $? -ne 0 ]]; then
-    fail "Failed to connect to metadata.google.internal."
-  fi
+  for i in $(seq 1 20) ; do
+    ping -q -w 10 metadata.google.internal && return 0
+    status "Waiting for connectivity to metadata.google.internal."
+    sleep $((i**2))
```

Comment:
If I understood it correctly, the total wait can be up to 1+4+9+16+....+400, which is too long

Reply:
It's the exact same logic as post_translate_test.sh. The only difference between the two scripts is the check for the 2 data disks.

Why is it too long?

Reply:
The sum is 2870, which is 40+min. Is it worth waiting for so long for internal service recovery?

Reply:
This will only run in our E2E tests, so I don't see an issue for waiting that long. Whether it's too long, maybe.

FWIW, this exact code was already reviewed and merged 6 months ago: https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1153

Reply:
Sounds good. Let's keep them sync, and we can discuss whether it's proper separately.